lib/active_admin_sidebar/activeadmin_views_pages_base.rb

```diff
@@ -31,6 +17,15 @@ def right_sidebar?
      !left_sidebar?
   end
 
+  def main_content_classes
+    classes = Arbre::HTML::ClassList.new
+    if skip_sidebar?
+      classes << 'without_sidebar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.